zerver/forms.py

```diff
@@ -208,6 +216,20 @@ def clean_email(self) -> str:
         if realm.is_zephyr_mirror_realm:
             email_is_not_mit_mailing_list(email)
 
+        if settings.BILLING_ENABLED:
+            try:
+                check_another_seat_can_be_added(realm)
+            except LicenseLimitError:
+                message = f"Signup of {email} failed because of license limit. Please increase the licenses or deactivate inactive users."
```

Comment:
This is sent to the signup notifications private stream. 

![Screenshot from 2021-05-25 16-34-20](https://user-images.githubusercontent.com/7190633/119487517-1ec64400-bd77-11eb-8f3e-5f408718d055.png)


In the screenshot, you can see that the messages changed slightly since I tweaked them a bit now.

Reply:
How about this:

"A new member ({email}) was unable to join your organization because all Zulip licenses for your organization are in use. Please [increase the number of licenses](link) or [deactivate inactive users](link) to allow new members to join."

Reply:
I suppose ideally we'd send warnings to the stream when there are, e.g., 3/2/1/0 licenses left. It would be better to add licenses then than when there's already a problem.